exercise13.py

```diff
@@ -0,0 +1,24 @@
+#https://edabit.com/challenge/6JNHBeGxY8dhTaPhs
+def collatz(a, b):
+    steps_a = 0
+    while a!=1:
+        if a %2 == 0:
+            a = a/2
+        else: 
+            a = 3*a + 1
+        steps_a +=1
+    steps_b = 0
+    while b!=1:
```

Comment:
this block looks identical to the previous block. Consider extracting it to a function